YandexCheckoutPaymentsApi/Source/Models/PaymentOptionInstrument/PaymentInstrumentYandexMoneyLinkedBankCard.swift

```diff
@@ -64,7 +64,7 @@ public class PaymentInstrumentYandexMoneyLinkedBankCard: PaymentOptionYandexMone
     ///   - cardType: Type of Bank card.
     ///   - identificationRequirement: Required type of user identification.
     public init(paymentMethodType: PaymentMethodType,
-                confirmationType: ConfirmationType?,
+                confirmationType: [ConfirmationType]?,
```

Comment:
Please, use plural form inn this case

Reply:
fxd in https://github.com/yandex-money/yandex-checkout-payments-api-swift/pull/7